projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -315,6 +323,151 @@ private boolean answer(String questionTemplateName, String paramsLine,
       return execute(wItemAs, outWriter);
    }
 
+   /**
+    * Validate the json-encoded {@code value} matches the valid setting
+    *  specified by {@code variable}.
+    *  <p>If {@code value} does not match the valid setting,
+    *  {@link BatfishException BatfishException} will be thrown.</p>
+    */
+   static void validate(JsonNode value, Variable variable)
+         throws BatfishException {
+      if (variable.getType().getStringType() &&
```

Comment:
Add a comment why this block is here. Add a comment before the `JSON_PATH` line explaining why that line is there and that you think this is a bug.